met-web/src/components/layout/SideNav/SideNav.tsx

```diff
@@ -28,40 +40,142 @@ const DrawerBox = () => {
+                >
+                    <ListItemButton
+                        onClick={() => setOpen(false)}
+                        sx={{
+                            justifyContent: 'flex-end',
+                            color: Palette.text.primary,
+                            verticalAlign: 'middle',
+                        }}
+                    >
+                        <FontAwesomeIcon icon={faArrowLeft} style={{ paddingRight: '0.75rem' }} />
+                        Close Menu
+                    </ListItemButton>
+                </ListItem>
+                <Divider sx={{ backgroundColor: colors.surface.gray[30] }} />
+            </>
+        );
+    };
+
     return (
         <Box
             sx={{
+                mr: isMediumScreen ? '1.25rem' : '0',
                 overflow: 'auto',
-                height: '100%',
-                background: Palette.primary.main,
+                backgroundColor: Palette.background.default,
+                borderTopRightRadius: '16px',
+                borderBottomRightRadius: '16px',
+                boxShadow: '0 5px 10px rgba(0, 0, 0, 0.4)',
+                mt: '5.625rem',
             }}
         >
-            <List sx={{ paddingTop: '2.5em' }}>
+            <List sx={{ pt: '0', pb: '0' }}>
+                {!isMediumScreen && <CloseButton />}
                 {filteredRoutes.map((route) => (
-                    <ListItem key={route.name}>
-                        <ListItemButton
-                            data-testid={`SideNav/${route.name}-button`}
-                            onClick={() => navigate(route.path)}
-                        >
-                            <When condition={currentBaseRoute === route.base}>
-                                <MetHeader4 color={Palette.secondary.main} bold>
-                                    {route.name}
-                                </MetHeader4>
-                            </When>
-                            <Unless condition={currentBaseRoute === route.base}>
-                                <MetHeader4 color={'white'}>{route.name}</MetHeader4>
-                            </Unless>
-                        </ListItemButton>
-                    </ListItem>
+                    <>
+                        <When condition={currentBaseRoute === route.base}>{handleListItems(route, 'selected')}</When>
+                        <Unless condition={currentBaseRoute === route.base}>{handleListItems(route, 'other')}</Unless>
+                    </>
                 ))}
-                <Divider sx={{ backgroundColor: 'var(--bcds-surface-background-white)' }} />
-                <UserGuideNav />
             </List>
         </Box>
     );
 };
 
-const SideNav = ({ open, setOpen, isMediumScreen, drawerWidth = 280 }: SideNavProps) => {
+const SideNav = ({ open, setOpen, isMediumScreen, drawerWidth = 300 }: SideNavProps) => {
     if (!drawerWidth) return <></>;
     return (
         <ThemeProvider theme={DarkTheme}>
```

Comment:
The DarkTheme provider around this component no longer really makes sense and should be removed, since the menu has gone from a dark background to a light one. This will probably save you some unnecessary style overrides deeper in the component.

Reply:
Thanks, I removed it.